www/src/components/gutters.js

```diff
@@ -0,0 +1,7 @@
+import presets from "../utils/presets"
+import { rhythm } from "../utils/typography"
+
+export const vP = rhythm(presets.gutters.default)
```

Comment:
@KyleAMathews gutters.js only exist because I ran into an error when trying to import rhythm right in `utils/preset` – there it got me a "rhythm is not a function". I believe I found a similar issue  … yes, here it is: https://github.com/gatsbyjs/gatsby-starter-default/issues/27 … Gotta fix those silly constant names btw.